resources/metric-ui.mjs

```diff
@@ -239,8 +239,12 @@ function renderScatterPlot({ values, width = 500, height, trackHeight, xAxisPosi
     // Axis + labels height:
     const axisHeight = 18;
     const axisMarginY = 4;
-    const markerSize = 5;
     const trackMargin = 2;
+    let markerSize = 5;
+    // Auto-adjust markers to [2px, 5px] for high iteration counts:
```

Comment:
I think this comment is outdated from a previous implementation, now it's `[1px, 5px]`

Reply:
that said I find 1px is a bit small. Maybe adjusting the formula to use 2px as the minimum would be better. What do you think?
(but my screen has a high dpi without zoom, so maybe that's why).

Reply:
Switched back to min size = 2px 👍